test/spectral_transform.jl

```diff
@@ -98,7 +129,7 @@ end
             for Grid in (   FullGaussianGrid,
                             FullClenshawGrid,
                             OctahedralGaussianGrid,
-                            OctahedralClenshawGrid,)
+                            OctahedralClenshawGrid)
                             # HEALPixGrid)
                             
                 P = Parameters(;NF,trunc,model=:shallowwater)
```

Comment:
This line still has `;Grid` missing... at the moment it just tests the default grid over and over again. But somehow there's some errors thrown otherwise, sounds like something for another issue/pull request though...

Reply:
Thank you for letting me know. I didn't notice this. So I was just looping over the same code...